posts/2021-11-03-tensorflow-gpu.rst

```diff
@@ -41,8 +41,15 @@ You can now select between GPU enabled (default) and CPU packages using the `ten
     # OR
     conda install tensorflow-gpu -c conda-forge
 
-When installing the `tensorflow` package, the package resolution will now default to the `tensorflow-gpu` package (that will also work on CPU). If you want the slimmer "cpu-only" package, then you can install `tensorflow-cpu` directly.
+When installing the `tensorflow` package, the package resolution will now default to the `tensorflow-gpu` package if the local machine has a GPU. Note that GPU-enabled packages can also work on CPUs, but one would need to override the enviornment variable `CONDA_CUDA_OVERRIDE` like below. This could be handy if you are in a situation where your current node (e.g. login node) on an HPC does not have GPUs, but the compute nodes with GPUs do not have interenet access.
 
+.. code-block:: bash
+
+    CONDA_CUDA_OVERRIDE="11.2" conda install tensorflow -c conda-forge
```

Comment:
I think this is bad advice for the average user. The CUDA version will outdate quickly and why would they want to hard code the CUDA version?

Reply:
@wolfv, see the info above. i.e. login nodes usually don't have GPUs, only the compute nodes do. Login nodes are where the interactive stuff like installing software is done.

Reply:
OK, fine. I didn't see that we had more text above :)